spec/models/member_spec.rb

```diff
@@ -180,5 +180,16 @@
         expect(member.monthly_worked_hours(Date.current)).to eq family_member.monthly_worked_hours(Date.current)
       end
     end
+
+    context 'when a member is enrolled on an event' do
+      let(:member) { create :member }
+
+      it "The event's duration are not counted in worked hours" do
```

Comment:
"#monthly_worked_hours when a member is enrolled on an event `ignores these hours`"
The convention is to use a verb right after `it`. You can try `rspec --format documentation` to get a feel of how you tests read as a full sentence

Reply:
done